bigbluebutton-html5/imports/api/ping-pong/server/publishers.js

```diff
@@ -3,7 +3,11 @@ import Logger from '/imports/startup/server/logger';
 import _ from 'lodash';
 
 const COLLECTION_NAME = 'ping-pong';
-const POLL_INTERVAL = 15000;
+const INTERVAL_IN_SETTINGS = (Meteor.settings.public.pingPong.clearUsersInSeconds) * 1000;
+const INTERVAL_TIME = INTERVAL_IN_SETTINGS < 10000 ? 10000 : INTERVAL_IN_SETTINGS;
+const PONG_INTERVAL_IN_SETTINGS = (Meteor.settings.public.pingPong.pongTimeInSeconds) * 1000;
+const PONG_INTERVAL = PONG_INTERVAL_IN_SETTINGS >= INTERVAL_TIME
```

Comment:
Shouldn't this be `const PONG_INTERVAL = PONG_INTERVAL_IN_SETTINGS >= INTERVAL_TIME / 2`. Or just switch them to Math.max/min.